src/main/java/org/rumbledb/api/Rumble.java

```diff
@@ -68,6 +69,8 @@ public SequenceOfItems runQuery(String query) {
             e.initCause(ex);
             throw e;
         }
+        String[] args = new String[0];
+        VisitorHelpers.resolveDependencies(mainModule, new RumbleRuntimeConfiguration(args));
```

Comment:
Can we extract new RumbleRuntimeConfiguration(new String[0]) to a variable that highlights the fact that this is just a placeholder configuration with no effects.

Reply:
DONE